ooni/ui/cli.py

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
I do not fully understand the difference in terms of taxonomy between `ooni/foo/cli.py` and `ooni/script/foo.py`. Is the former the implementation and the latter just the "main"?